pages/projects/myprojects.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { useEffect, useState } from "react";
+import commonStyles from "../../styles/common.module.scss";
+import styles from "./Projects.module.scss";
+import Head from "next/head";
+import { Pagination, Typography } from "@equinor/eds-core-react";
+import { Layouts } from "../../layouts/LayoutWrapper";
+import { ProjectListSection } from "../../components/ProjectListSection";
+import { useQuery } from "react-query";
+import { getProjects } from "../../services/projectApi";
+import { useAccount, useMsal } from "@azure/msal-react";
+import { getUserShortName } from "../../utils/getUserShortName";
+import SideNavBar from "components/SideNavBar";
+import SortMenu from "components/SortMenu";
+
+const itemsPerPage = 16;
+export default function Projects(): JSX.Element {
+  const { accounts } = useMsal();
+  const account = useAccount(accounts[0]);
+  const userNameFilter = getUserShortName(account);
+  const [page, setPage] = useState(1);
+  const [orderBy, setOrderBy] = useState("name");
+
+  const { data, isLoading, error } = useQuery(
+    ["projects", page, userNameFilter, orderBy],
```

Comment:
Remember this query is unique. So instead of using "projects" as the first key. It would make sense to call it "myProjects" to enable local caching. Also for the other pages.